### ISSUE TITLE
Start page/guidance content review and gap analysis

### ISSUE THREAD

**Issue by paris-dp** [digital-land/submit#395]
**Context:** The journey in to the service requires improvement. We need to understand what information is presented to users at present, and where user needs are not being met. 

Discussed in Design time on 05/09/24. Agreed that a content review/gap analysis was required. 

**[Artefact which may be useful:](https://app.mural.co/t/mhclg2837/m/mhclg2837/1728654025264/16e0efd10d44506f2e66e5b02de469a99692b024?wid=0-1724342031813)**

**Comment by paris-dp** [comment]
Discussed with Gwen on 24.09: gap analysis to be conducted

**Comment by maddie-broxup** [comment]
Paris, Gwen, Alex and I chatted about this during our content / UX meeting. 

Gwen and I will work on a high level audit of the service pages. We'll:

- create a spreadsheet to capture the work
- scrape the URLs (Gwen / Colm to do this)
- label the content types
- create a rationale for archiving / deleting / moving pages

Maddie and Gwen to move this forwards and start by creating a spreadsheet.

**Comment by maddie-broxup** [comment]
Adding this comment based on the sprint review we just did in providers team. 

Gwen has said Colm can't find the URL scrapes so Adam is picking this up. 

In the meantime, Maddie to begin with an audit of:

- step by step guidance
- guidance pages
- BFL guidance 

We observed a user in research yesterday getting lost in the guidance, navigating to it from the step by step pages and not being able to find her way back to the service. We want to look at how we might improve this experience for users, whilst also reviewing existing content across all pages for consistency and duplication.

**Comment by maddie-broxup** [comment]
We've received the URL scrapes from Adam, in this spreadsheet: https://docs.google.com/spreadsheets/d/1IjuRyybaxZznQI29eEjUCSiTRegkYtgEi-Guv5umaBU/edit?usp=sharing

Gwen and I will discuss how we go about tackling the audit

**Comment by maddie-broxup** [comment]
This has become more of a BFL-focussed piece of work as priority, until we can find the time to do a bigger audit of the guidance pages. 

We've agreed that Gwen will include a link to the service from the top of the BFL guidance page on gov.uk in the first instance. Gwen has access to Whitehall publisher (which I don't!), and already has a ticket in her team's sprint for this.

**Comment by maddie-broxup** [comment]
Link to service has been added to BFL guidance page by Gwen - https://www.gov.uk/government/publications/brownfield-land-registers-data-standard/publish-your-brownfield-land-data 

Also in the process of putting together a proposal for an away day with UX folks (possibly in the new year now) to look at the guidance pages holistically: https://docs.google.com/document/d/1HdkDIIfZT0UcR-zL-KPdJLx0sY_kGI74FTc1MOk_Q58/edit?usp=sharing

**Comment by maddie-broxup** [comment]
Guidance away day proposal shared with Charlie for comments

**Comment by paris-dp** [comment]
Met with Maddie and Gwen today to discuss initial approach to reviewing pages ahead of a wider team away day review. Notes added to [board](https://app.mural.co/t/mhclg2837/m/mhclg2837/1728654025264/16e0efd10d44506f2e66e5b02de469a99692b024?wid=0-1732267576943). 

Agreed actions: Gwen and Maddie to work on an initial draft. Paris to sense check following this.

**Comment by paris-dp** [comment]
This [blog](https://mhclgdigital.blog.gov.uk/2024/12/03/digital-planning-applying-digital-principles-and-innovation-to-transform-englands-planning-system/) may be useful when working content about the the programme/programme's mission

**Comment by maddie-broxup** [comment]
Initial sift of guidance pages and recommendations on which bits to remove / repurpose to landing page: https://docs.google.com/document/d/1i3_9ofoFxwJXBMjQeR5gA-wsdb0hC8gqVBcFKQr-zaw/edit?pli=1&tab=t.0#heading=h.lro9wtkte5n6

Gwen and Maddie to continue working on drafting landing page.

Proposing guidance away day for 30th Jan but still waiting to hear back from some people first - will confirm when it's been agreed

**Comment by maddie-broxup** [comment]
Guidance away day confirmed for 30th Jan and communicated to attendees. Working on the session for the day: https://app.mural.co/t/mhclg2837/m/mhclg2837/1735815762226/4e07ba02ab6f83001b9d95f98752993f46e715bc

Meeting with Paris, Alex and Gwen tomorrow to firm up plans

**Comment by maddie-broxup** [comment]
Agreed with rest of team we'd close this now and start fresh with new tickets under [the same epic](https://github.com/digital-land/submit/issues/807).
